nsv13/code/modules/jobs/faction.dm

```diff
@@ -178,6 +179,7 @@
 	name = JOB_NAME_SECURITYOFFICER + " (SolGov)"
 	uniform = /obj/item/clothing/under/ship/solgov/engsec
 	accessory = /obj/item/clothing/accessory/solgov_jacket/engsec
+	suit_store = /obj/item/gun/energy/laser/retro/old
```

Comment:
This is a gun for use on ruins, you shouldn't be using the /old subtype, and I question even the use of the /retro type in the first place. Why not just give them a working normal energy gun like the ones actually in the armory?

Reply:
I use retro ones since they're logistically contemporary here, and because they fit the aesthetic of solgov. I'm using the old variant because it has fewer shots(it's also what's in the squad vendors), since I'm a little bit hesitant to replace a non lethal weapon with what is essentially a full laser gun
I'm also hesitant to add e-guns to sec officers roundstart loadout for the obvious reasons